pkg/gotohelm/helmette/shims.go

```diff
@@ -114,6 +115,8 @@ func Unwrap[T any](from Values) T {
 			switch reflect.New(to).Interface().(type) {
 			case *resource.Quantity:
 				return valuesutil.UnmarshalInto[*resource.Quantity](val)
+			case *corev1.Volume:
```

Comment:
Oh dang, I bet we've got some other types that will bite us. Can you add a regression test for this?